app/config.py.example

```diff
@@ -6,6 +6,8 @@ from fastapi_mail import ConnectionConfig
 
 # DATABASE
 DEVELOPMENT_DATABASE_STRING = "sqlite:///./dev.db"
+# Set the following True if working on PSQL environment or set False otherwise
+PSQL_ENVIRONMENT = False
```

Comment:
Maybe you want to add it as env value?

Reply:
I considered it and chose otherwise, thank you!

Reply:
I'd be happy to hear why :)